testnet/docker-compose.yml

```diff
@@ -24,25 +26,43 @@ services:
       "--enclaveRPCAddress=enclave:11000",
       "--rollupContractAddress=$MGMTCONTRACTADDR",
       "--privateKey=$PKSTRING",
-      "--clientRPCHost=0.0.0.0"
+      "--clientRPCHost=0.0.0.0",
     ]
 
   enclave:
+    privileged: true
+    volumes:
+      - /dev/sgx:/dev/sgx
     networks:
       - default
     environment:
       MGMTCONTRACTADDR: some_address
       ERC20CONTRACTADDR: some_address
       HOSTID: some_address
+      OE_SIMULATION: 0
     image: testnetobscuronet.azurecr.io/obscuronet/obscuro_enclave:latest
     entrypoint: [
                  "ego", "run", "/home/obscuro/go-obscuro/go/enclave/main/main",
                  "--hostID=$HOSTID",
                  "--address=:11000",
-                 "--useInMemoryDB=false",
-                 "--sqliteDBPath=/data/sqlite.db",
                  "--managementContractAddress=$MGMTCONTRACTADDR",
                  "--erc20ContractAddresses=$ERC20CONTRACTADDR,$ERC20CONTRACTADDR",
                  "--viewingKeysEnabled=false",
-                 "--hostAddress=host:10000"
-    ]
\ No newline at end of file
+                 "--hostAddress=host:10000",
+                 "--willAttest",
+                 "--useInMemoryDB=false",
+                 "--edgelessDBHost=edgelessdb"
+    ]
+
+  edgelessdb:
+    privileged: true
+    volumes:
+      - /dev/sgx:/dev/sgx
+    networks:
+      - default
+    environment:
+      EDG_EDB_CERT_DNS: edgelessdb
+    ports:
+      - "3306:3306"
+      - "8080:8080"
+    image: ghcr.io/edgelesssys/edgelessdb-sgx-1gb:latest
```

Comment:
There's also a 4GB version of this (same path but with "4-gb" at the end. Maybe worth using that to buy us a bit more time before we have to think about archiving and optimisation or wait til it becomes a problem?